src/main/java/at/hannibal2/skyhanni/test/command/TrackSoundsCommand.kt

```diff
@@ -0,0 +1,50 @@
+package at.hannibal2.skyhanni.test.command
+
+
+import at.hannibal2.skyhanni.events.PlaySoundEvent
+import at.hannibal2.skyhanni.utils.DelayedRun
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.OSUtils
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.util.concurrent.atomic.AtomicBoolean
+import kotlin.time.Duration.Companion.seconds
+
+object TrackSoundsCommand {
+
+    var cutOfTime: SimpleTimeMark = SimpleTimeMark.farPast()
+
+    val sounds = mutableListOf<PlaySoundEvent>()
+
+    var enable = AtomicBoolean(true)
+
+    fun command(args: Array<String>) {
+        if (!enable.get()) {
+            LorenzUtils.chat("Still tracking sounds, wait for the other tracking to complete before starting a new one")
+            return
+        }
+        enable.set(false)
```

Comment:
`enable` is a pretty bad name for this, i think. Is enable the command being enabled or the listening being enabled? Maybe use something like `currentlyRecording`.